#Standardized message for the communication of intersection geometry to the vehicle

std_msgs/Header header
int64 vehicle_id
nav_msgs/Path path

